test/fit_path/__init__.py

```diff
@@ -0,0 +1,5 @@
+import sys
```

Comment:
Rather than using git (and depending on an outside tool) we could do:
from os import dirname, abspath
_test_root = dirname(abspath(__file__))
sys.path.append(os.path.join(_test_root, 'test')
sys.path.append(os.path.join(_test_root, 'test', 'common')

(not using git also helped when I was doing some vagrant stuff where I only wanted to share the test directory (and NOT all of rackhd) into the image for doing test development, and of course the git command didn't work since the git-top-level didn't actually exist inside the image).